2_drivers.yml

```diff
@@ -9,21 +9,44 @@ sources:
 targets:
   2_observations:
     depends:
-      #- out_data/placeholder_for_gefs_archive
-      #- out_data/placeholder_for_nldas
-      #- out_data/reservoir_io.csv
+      - out_data/NLDAS_time[0.377900]_x[397]_y[136].csv
+      - out_data/NLDAS_time[0.377900]_x[401]_y[136].csv
+      - out_data/noaagefs_meteo_driver_files.zip
+      - out_data/reservoir_io_obs.csv
+      - out_data/reservoir_io_sntemp.csv
       
-  # gefs data will come from 
-  #out_data/placeholder_for_gefs_archive:
-   # command: copy_files()
-    
-  #out_data/palceholder_for_nldas:
-   # command: copy_files()
-   
+  # NLDAS Files
+  ## NLDAS_time[0.377900]_x[397]_y[136].csv
+  out_data/NLDAS_time[0.377900]_x[397]_y[136].csv:
```

Comment:
Decided to copied these to out_data before any further data release manipulation. I think in last year's release they were directly added to `remake.yml`. Is this ok? Would be fine if we skip this step.